versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5846 @@
+
+Every WDL implementation must support the ability to output this standard output. It is suggested that WDL implementations make the standard format be the default output format.
+
+For example, given this workflow:
+
+```wdl
+workflow example {
+  ...
+  output {
+    String foo = cafeteria.inn
+    File analysis_results = analysis.results
+    Int read_count = readcounter.result
+    Float kessel_run_parsecs = trip_to_space.distance
+    Boolean sample_swap_detected = array_concordance.concordant
+    Array[File] sample_variants = variant_calling.vcfs
+    Map[String, Int] droids = escape_pod.cargo
+  }
+}
+```
+
+The output JSON will look like:
+
+```json
+{
+  "example.foo": "bar",
+  "example.analysis_results": "/path/to/my/analysis/results.txt",
+  "example.read_count": 50157187,
+  "example.kessel_run_parsecs": 11.98,
+  "example.sample_swap_detected": false,
+  "example.sample_variants": ["/data/patient1.vcf", "/data/patient2.vcf"],
+  "example.droids": {"C": 3, "D": 2, "P": 0, "R": 2}
+}
+```
+
+It is recommended (but not required) that JSON outputs be "pretty printed" to be more human-readable.
+
+## Specifying / Overriding Runtime Attributes
+
+In addition to specifying workflow input in JSON, the user can also specify (or override) runtime attributes. Input values for standardized runtime attributes must adhere to the [supported types and formats](#runtime-section). Runtime values provided as inputs always supersede values supplied directly in the WDL. Any runtime attributes that are not supported by the execution engine are ignored.
+
+To differentiate runtime attributes from task inputs, the `runtime` namespace is added after the task name. For example:
+
+```json
+{
+  "wf.t1.runtime.memory": "16 GB",
+  "wf.t2.runtime.cpu": 2,
+  "wf.t2.runtime.disks": "100",
+  "wf.t2.runtime.container": "mycontainer:latest"
+}
+```
+
+## JSON Serialization of WDL Types
+
+### Primitive Types
+
+All WDL types serialize naturally to JSON values:
+
+|WDL Type        |JSON Type|
+|----------------|---------|
+|`Int`           |number   |
+|`Float`         |number   |
+|`Boolean`       |boolean  |
+|`String`        |string   |
+|`File`          |string   |
+|`None`          |null     |
+
+JSON has a single numeric type - it does not differentiate between integral and floating point values. A JSON `number` is always deserialized to a WDL `Float`, which may then be coerced to an `Int` if necessary.
+
+JSON also does not have a specific type for filesystem paths, but a WDL `String` may be coerced to a `File` if necessary.
+
+### Array
+
+Arrays are represented naturally in JSON using the `array` type. Each array element is serialized recursively into its JSON format.
+
+When a JSON `array` is deserialized to WDL, each element of the array must be coercible to a common type.
+
+### Struct and Object
```

Comment:
This seems out of order.  Better to keep the order in the text as array, pair, map then struct (leave out Object in header but retain in text with trash can). 

Also, I'm thinking you're going to want to rephrase this text too now that `object` as a WDL type is being deprecated, and becuase there are lots of references to `o/Bject`s here but in multiple contexts.  What will this text mean once `object` is gone?  How are `struct`s going to be represented in JSON?  

I think you mean something like, "`Struct`s (and the to be deprecated `Object`) themselves are represented naturally by JSON `object`s.   Each WDL `struct` (and also `Object`) member value is serialized recursively into its specific JSON type.  "

Then I don't know that I'm understanding more from the next sentence.  

"When a JSON `object` is deserialized, each value is deserialized to its most likely WDL type. " Then, state how JSON `object`s will be coerced into a `struct`.  Will this be only if the struct you want is specified?  Must each element of the JSON `object` be coercible to the type specified in the `struct` definition?  Leave out mention of `Object` which won't be a thing.